### ISSUE TITLE
Add "today" button to report date picker

### ISSUE THREAD

**Issue by fniessink** [ICTU/quality-time#378]
# User story

As a user I want to have a today button in the date picker so that I don't have to look up the date somewhere else to select today.

## Acceptance criteria

- [x] Clicking the today button in a date picker selects the current date.

## Assumptions

- Either the current date picker allows for this functionality or a composite component can be made.

## Open issues

- Should selecting today also close the date picker and save the change?

## Out of scope

- Upgrading the date picker, see #4167.
- Other suggestions made below in the comments.

# Tasks

### Frontend

- [x] Add today button to date picker.

## Documentation

- [x] Update changelog

## Related issues

- Blocked by: #4467.
- #4167

**Comment by Sebastiaan127001** [comment]
%VOTE

**Comment by Sebastiaan127001** [comment]
And also mark the current date in the calendar
and have it start on Monday as the first day of the week.

**Comment by Sebastiaan127001** [comment]
and another related feature request: 

- [ ] add days left after the set date. so in the below image that would be 2022-04-20 (6 days from now)
![image](https://user-images.githubusercontent.com/95095453/178928929-189aea14-23d0-443f-931c-7b448ea4c25a.png)

- [ ] change title 'technical debt end date' to 'technical debt end'
- [ ] mark the end date of sprint when QT knows this

**Comment by Sebastiaan127001** [comment]
And if the user clicks on a greyed out date from the next or previous month, the picker should switch to that month.
